cumulus/tasks/discover-pdrs/index.js

```diff
@@ -17,78 +16,60 @@ const local = require('@cumulus/common/local-helpers');
 function discoverPdrs(event) {
   try {
     const config = get(event, 'config', {});
-    const queue = get(config, 'useQueue', true);
```

Comment:
Why are you removing all of the Queue stuff?

Reply:
When I added output schema validation, I realized that we were returning completely different responses depending on whether or not queuing was used.  Not only was that causing the tests to now fail, it was also confusing for people trying to use that task.  I talked it over with @scisco and we decided that:

1) The discover tasks were doing to too much.  They were both discovering granules or PDRs, but they also contained the logic to then enqueue those items for ingest.
2) There are already `queue-granules` and `queue-pdrs` tasks which do a fine job of enqueuing items to be ingested.
3) We should remove the queuing functionality from the discovery tasks, simplifying those tasks.

So, that's why all of the queuing functionality has been removed from granule and PDR discovery.  And also a big part of reason that this PR is gigantic.